test/unit/specs/store/governance/votes.spec.js

```diff
@@ -60,26 +60,27 @@ describe(`Module: Votes`, () => {
       }
     }
     let dispatch = jest.fn()
-    proposals.forEach(async (proposal, i) => {
+    const proposalIds = Object.keys(proposals)
+    proposalIds.forEach(async (proposal_id, i) => {
```

Comment:
same

Reply:
Here I kept it because I needed the length in the loop on line 81